MAUI/Segmented-Control/customization.md

```diff
@@ -0,0 +1,395 @@
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the segment width
+You can use `SegmentWidth` property of `SfSegmentedControl`, to customize the segment width of the segmented control.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl"
+                                SegmentWidth="100">
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SegmentWidth = 100;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the segment height
+You can use `SegmentHeight` property of `SfSegmentedControl`, to customize the segment height of the segmented control.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl"
+                                SegmentHeight="40">
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.SegmentHeight = 40;
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the segment text style
+You can use `TextStyle` property of `SfSegmentedControl`, to customize the text style of the segmented control.
+
+{% tabs %}
+{% highlight XAML %}
+
+<ContentPage
+    xmlns:buttons="clr-namespace:Syncfusion.Maui.Buttons;assembly=Syncfusion.Maui.Buttons">
+    <buttons:SfSegmentedControl x:Name="segmentedControl">
+        <buttons:SfSegmentedControl.TextStyle>
+            <buttons:SegmentTextStyle TextColor="Red" FontAttributes="Italic" FontSize="20"/>
+        </buttons:SfSegmentedControl.TextStyle>
+    </buttons:SfSegmentedControl>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+using Syncfusion.Maui.Buttons;
+. . .
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        SfSegmentedControl segmentedControl = new SfSegmentedControl();
+        segmentedControl.TextStyle = new SegmentTextStyle()
+        {
+            TextColor = Colors.Red,
+            FontAttributes = FontAttributes.Italic,
+            FontSize = 20,
+        };
+        this.Content = segmentedControl;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing the segment item
```

Comment:
Customizing the segment control each item

Reply:
Heading changed